mergify_engine/github_events.py

```diff
@@ -35,6 +36,15 @@
 LOG = daiquiri.getLogger(__name__)
 
 
+async def store_active_users(
+    redis_cache: utils.RedisCache, repo: github_types.GitHubRepository
+) -> None:
+    id_owner = repo["owner"]["id"]
+    id_repo = repo["id"]
+    await redis_cache.hset(id_owner, "timestamp", time.time())
+    await redis_cache.hset(id_owner, "repo_id", id_repo)
```

Comment:
There's a couple of issues here:
- this is going to set a key named something like `12345` into Redis, which is not really great: there's plenty of keys here, a key named `12345` does not really help to understand what it stores. You should names your keys with something like `active_users~{owner_id}`
- Storage works. Now, you also have to think about query time. So what's the typical query you'll get? I can tell you, it's "give me the list of users we got active over the last month for this organization".
Here are a few hints: first problem, you don't store the organization. Second problem, your data structure here does not help you answering the query efficiently.